### DISCUSSION TITLE
A simple way to set up WiFi on headless devices

### DISCUSSION THREAD

**Discussion by dirkhh** [MichaIng/DietPi#6542] [Q&A] [answered]
My use case revolves around building simple to use images for people who really don't know a lot about computers. So ideally I'd love to have them write an image to SD card and then make it as easy as possible to bring up the device. Which is challenging if it is a WiFi only device - unless there is a fat (fat32/vfat/extfat) /boot partition that makes it reasonably easy for people to make the four small edits to two files (ugh... it would be so much easier to explain if it was all in the same file). But on many non-RPi images there is only one ext4 partition. Which most non-technical people have no way of mounting / editing files on.
Am I missing an option that would make this possible?

**Comment by MichaIng** [comment, marked as answer]
Ah this is the question you were referring to. Sorry for missing it.

I think we answered/discussed the problem and your idea to solve it already on the other thread: https://github.com/MichaIng/DietPi/issues/6570#issuecomment-1697899163
